app/src/main/res/layout/activity_main.xml

```diff
@@ -24,8 +24,8 @@
             android:layout_width="match_parent"
             android:layout_height="wrap_content"
             android:autofillHints="phone"
-            android:inputType="textMultiLine"
-            android:minLines="3" />
+            android:inputType="phone"
+            android:lines="1" />
```

Comment:
Thanks for taking the initiative @cyb3rko and putting up a detailed PR with screenshots, I appreciate it! :)

I'm happy to accept the messenger app icon changes as they are, but I have thoughts on the input text field commit. The reason that it has the `textMultiLine` inputType + a minimum height of 3 lines is because I'd like to support not just manual input but also clicking on SMS links or sharing a block of text from a website or message into the app and letting it extract the phone numbers.

[launch-chat-text-share.webm](https://github.com/user-attachments/assets/5a4f0a2e-8f4a-45ab-b75b-ef24a65c45d8)

How do you propose the app handle the contradictory needs of the numerical keyboard input layout + single line for manual entry vs text input layout + multiple lines for shared text? One option would be to introduce a settings page with a setting that programmatically controls the input type, or a toggle next to this input field itself.

If you'd like to move forward with the app icon changes, I propose carving that commit out into a separate PR.

Reply:
I think the phone number field should not contain text at any time.  
My idea would be to be able to paste this whole block of text by clicking the already existing clipboard button next to the input field.

Instead of just pasting the text we could already extract and select the right number there instead of doing that right before sending.

What do you think?

Reply:
> I think the phone number field should not contain text at any time.

This removes the currently supported user journey of
1. Come across a block of text on a website or on a different app, something like "You can reach us at 111222333"
2. The user selects either the full paragraph (if it's a text message for example) or a few lines of text and uses the "Share" option to share with Launch Chat
3. The user would like to modify the phone number in the text, perhaps to add a country code (especially relevant if the number is not meant to be interpreted as the user's default country code, once default country codes are implemented), before they use one of the messenger action buttons

I have an alternative proposal in #45 that keeps this functionality while defaulting to "phone" as the input type of the phone number input field. Curious if you have any thoughts on that.

Reply:
I've gone ahead and merged #45.
If you could drop the second commit out of this PR, I'm happy to approve it.

Reply:
Done